prince/files/prince.init

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh /etc/rc.common
+# Init script for Prince
+# Copyright (C) 2007 OpenWrt.org
```

Comment:
Please update to 2018 the Copyright. Also check if there is a proper Licence header for files committed into OpenWrt.

Reply:
Corrected